packages/extension-webgazer/src/index.ts

```diff
@@ -0,0 +1,327 @@
+import { JsPsych, JsPsychExtension, JsPsychExtensionParameters } from "jspsych";
+
+// we have to add webgazer to the global window object because webgazer attaches itself to
+// the window when it loads
+declare global {
+  interface Window {
+    webgazer: any;
+  }
+}
+
+interface InitializeParameters extends JsPsychExtensionParameters {
```

Comment:
My thought was that `JsPsychExtensionParameters` might, in the future, have some shared property across extensions so it would be good to model this now. But admittedly I don't have any idea about what those parameters would be, and I thought of this prior to adding the extension-specific types in the webgazer extension. 

So I think it does make sense to just drop this entirely.